charts/rstudio-pm/templates/deployment.yaml

```diff
@@ -86,11 +98,6 @@ spec:
             readOnly: true
             mountPath: "/home/rstudio-pm/.aws"
           {{- end }}
-          {{- if .Values.rstudioPMKey }}
```

Comment:
Is there any case where a user might want to do this instead of an env var? I guess they can always do it with `pod.volumes` or `pod.volumeMounts`?

Reply:
No, if you are not running as root it's basically impossible to use it as a volume mount, and I can't think of a scenario where you would prefer a volume mount to an environment variable.